chapters/equations.tex

```diff
@@ -518,6 +519,11 @@ \subsection{assert}\label{assert}
        AssertionLevel.warning);
 assert(T > 200 and T < 500, "Medium model outside feasible region");
 \end{lstlisting}
+
+It is recommended that asserts have a simple message as above, formulated with the recommended tool behavior in mind.
+Tools are recommended to include the failed condition in the message.
+Tools are also (especially for literal strings) recommended to automatically include the values of relevant variables, as part of the log or interactively accessible.
+Writing \lstinline!assert(T<500, "Temperature = "+String(T)+" was above 500");! is thus not recommended, and is likely to lead to duplicated information -- although tools may have specific compatibility handling of such cases.
```

Comment:
It should be sufficient to say _likely_; just let tools to their "clever" thing without mentioning it in the specification.  Also remove punctuation at end of inline code:
```suggestion
Writing \lstinline!assert(T<500, "Temperature = "+String(T)+" was above 500")! is thus not recommended, and is likely to lead to duplicated information.
```